### ISSUE TITLE
Feature request: improved notifications (correct offline count & important reviews)

### ISSUE THREAD

**Issue by jeff8v7** [davidsansome/tsurukame#362]
The app notifications are one of the best features of tsurukame. Love them!  I have 2 ideas for improvement.

1) when offline, notifications may show the wrong total (if user studies using a different device during that time). Would be more accurate to say "+10 reviews" than "40 reviews" in this case.

2) would be great to see a different notification for important reviews.  For example, "Level up" reviews are notable. Also, the first review of items is notable (since it's more likely to be forgotten)

**Comment by turtlemaster19** [comment]
@jeff8v7 As for point number one, if a user does reviews or lessons on a different device, even plus ten reviews would be inaccurate.

As for point number two, I agree that including whether level-up or first SRS stage reviews are present would be a good idea.

**Comment by jeff8v7** [comment]
> if a user does reviews or lessons on a different device, even plus ten reviews would be inaccurate.

Good point. That’s true for some cases. Like, if the user‘s phone was offline for 5 hours, for example, and they did lessons elsewhere 4 hours ago, they’re would be some reviews tsurukame doesn’t know about. 

In general “+10 new reviews” would be inaccurate less often than “55 (total) reviews” while offline, but it wouldn’t always be accurate. 

I suppose, request 1 is an edge case anyway for offline notifications. I thought it was a bug until I realized I was causing it by taking my phone offline, then doing a bunch of reviews on my iPad before bed. 

When I woke up, my phone says “150 reviews” but when I open the app, they’re are only 35 reviews. Not a big deal...

**Comment by jeff8v7** [comment]
Hmm... an easier solution...
28 reviews available (10 new)
